taxes/get-tax-controller.js

```diff
@@ -0,0 +1,25 @@
+const Calculator = require('./calculator')
+const TaxRegulator = require('./tax-regulators')
+
+class GetTaxController {
+    process(ctx) {
+        const body = ctx.request.body;
+        if (!body.state || !body.productPrice) {
+            return ctx.body = { error: "Required parameters not present" };
```

Comment:
it's relatively strange to see return here, but more like stylistic issue. Do you know what you are returning here?